addons/html_builder/static/src/core/building_blocks/builder_list.js

```diff
@@ -38,6 +38,8 @@ export class BuilderList extends Component {
         defaultNewValue: { type: Object, optional: true },
         columnWidth: { optional: true },
         forbidLastItemRemoval: { type: Boolean, optional: true },
+        noAddItemButton: { type: Boolean, optional: true },
+        notEditable: { type: Boolean, optional: true },
```

Comment:
It s not a good patter to have a negation in the name of a var or a props.
hasAddItem
isEditable
(If you have a better name, you can propose :) )

Reply:
Maybe one props `isEditable` is enough for your use case.